code/modules/research/xenobiology/crossbreeding/destabilized.dm

```diff
@@ -0,0 +1,356 @@
+
+/obj/item/slimecross/destabilized/blue
+	colour = "blue"
+	effect_desc = "Wets the floor under the hit creature, causing them to slip."
+
+/obj/item/slimecross/destabilized/blue/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	var/datum/reagents/R = new/datum/reagents(50)
+	R.add_reagent(/datum/reagent/water, 50)
+	R.reaction(get_turf(A), TOUCH, 50)
+	qdel(R) //Reaction doesnt use up the reagents
+	if(iscarbon(A))
+		var/mob/living/carbon/C = A
+		C.slip(100, src, NONE, 20, FALSE)
+
+/obj/item/slimecross/destabilized/metal
+	colour = "metal"
+	effect_desc = "Traps the hit creature in a metal cage."
+
+/obj/item/slimecross/destabilized/metal/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		to_chat(L, "<span class='warning'>You suddenly find yourself trapped in a dark cage!</span>")
+		L.apply_status_effect(/datum/status_effect/cageTrapped)
+	else
+		new /obj/structure/metalCage(get_turf(A))
+
+/obj/item/slimecross/destabilized/yellow
+	colour = "yellow"
+	effect_desc = "Gives the hit creature an electric shock, or EMP's the tile."
+
+/obj/item/slimecross/destabilized/yellow/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		L.electrocute_act(rand(5,20), src, 1, TRUE)
+		playsound(L, "sparks", 50, 1)
+	else
+		empulse(get_turf(A), 1, 1)
+
+/obj/item/slimecross/destabilized/darkpurple
+	colour = "dark purple"
+	effect_desc = "Makes the hit creature highly flammable, or creates a plasma cloud."
+
+/obj/item/slimecross/destabilized/darkpurple/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		L.fire_stacks += 5
+		L.reagents.add_reagent("plasma", 10)
+	else
+		var/datum/reagents/R = new/datum/reagents(10)
+		R.add_reagent("plasma", 10)
+		var/datum/effect_system/smoke_spread/chem/smoke = new
+		smoke.set_up(R, 1, get_turf(A))
+		smoke.start()
+
+/obj/item/slimecross/destabilized/darkblue
+	colour = "dark blue"
+	effect_desc = "Freezes the hit creature in a block of solid ice."
+
+/obj/item/slimecross/destabilized/darkblue/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		L.apply_status_effect(/datum/status_effect/freon/destabilized)
+
+/obj/item/slimecross/destabilized/silver
+	colour = "silver"
+	effect_desc = "Feeds the hit creature an unhealthy amount."
+
+/obj/item/slimecross/destabilized/silver/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		L.nutrition = NUTRITION_LEVEL_FAT + 1
+		to_chat(L, "<span class='warning'>You suddenly feel gorged!</span>")
+
+/obj/item/slimecross/destabilized/bluespace
+	colour = "bluespace"
+	effect_desc = "Teleports the hit creature next to the thrower."
+
+/obj/item/slimecross/destabilized/bluespace/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		do_teleport(A, get_turf(thrower), 1)
+	if(isobj(A))
+		var/obj/O = A
+		if(!O.anchored)
+			do_teleport(O, get_turf(thrower), 1)
+
+/obj/item/slimecross/destabilized/sepia
```

Comment:
This extract has two instant stuns without counter play, the thing we're mostly trying to get rid of at the moment. Please put these on a delay.